training/movinet_with_hmdb51/Classify-Vanilla-trained.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c9b8f6e9-668c-4c4b-81dd-5fcfb7956d51",
+   "metadata": {},
+   "source": [
+    "# Classification with a vanilla model vs trained model.\n",
```

Comment:
```suggestion
    "# Activity Recognition on Videos Stored in ApertureDB\n",
```

Reply:
I am debating if it should address finetuninig in the title but I dont suppose we show trianing example, do we?